cypress/integration/brewery/CreateScenario-tests.spec.js

```diff
@@ -259,8 +270,124 @@ describe('Create scenario', () => {
 
     cy.get(SELECTORS.scenario.selectInput).find('input').should('have.value', scenarioChildName);
 
-    cy.get(SELECTORS.scenario.parameters.stockInput).find('input').should('have.value', childStock);
-    cy.get(SELECTORS.scenario.parameters.restockInput).find('input').should('have.value', childRestock);
-    cy.get(SELECTORS.scenario.parameters.waitersInput).find('input').should('have.value', childWaiters);
+    cy.get(SELECTORS.scenario.parameters.brewery.stockInput).find('input').should('have.value', childStock);
+    cy.get(SELECTORS.scenario.parameters.brewery.restockInput).find('input').should('have.value', childRestock);
+    cy.get(SELECTORS.scenario.parameters.brewery.waitersInput).find('input').should('have.value', childWaiters);
+  });
+
+  it('can create scenario, edit/discard parameters and switch between parameters tabs', () => {
+    // Log and launch app on scenario view
+    cy.visit(PAGE_NAME.SCENARIO);
+    cy.login();
+
+    // Create Scenario with some paramaters tabs
+    cy.get(SELECTORS.scenario.createButton).click();
+    cy.get(SELECTORS.scenario.createDialog.dialog).should('be.visible');
+    cy.get(SELECTORS.scenario.parameters.tabs).should('be.visible');
+
+    cy.get(SELECTORS.scenario.createDialog.masterCheckbox).click();
+    cy.get(SELECTORS.scenario.createDialog.nameTextfield).type(scenarioWithBasicTypes);
+    cy.get(SELECTORS.scenario.createDialog.datasetSelect).click().clear()
+      .type(DATASET.BREWERY_ADT + '{downarrow}{enter}');
+    cy.get(SELECTORS.scenario.createDialog.typeSelect).click().clear()
+      .type(SCENARIO_TYPE.BASIC_TYPES + '{downarrow}{enter}');
+
+    cy.intercept('POST', urlRegexWithoutSuffix)
+      .as('requestCreateScenario');
+    cy.intercept('GET', urlRegexWithoutSuffix)
+      .as('requestUpdateScenarioList');
+
+    cy.get(SELECTORS.scenario.createDialog.submitButton).click();
+
+    let scenarioCreatedId, scenarioCreatedName;
+    cy.wait('@requestCreateScenario').should((req) => {
+      scenarioCreatedName = req.response.body.name;
+      scenarioCreatedId = req.response.body.id;
+      cy.wrap(scenarioCreatedId).as('scenarioCreatedId');
+      expect(scenarioCreatedName).equal(scenarioWithBasicTypes);
+    });
+
+    cy.wait('@requestUpdateScenarioList').should((req) => {
+      const nameGet = req.response.body.find(obj => obj.id === scenarioCreatedId).name;
+      expect(nameGet).equal(scenarioCreatedName);
+      expect(nameGet).equal(scenarioWithBasicTypes);
```

Comment:
If you already checked that `scenarioCreatedName` is equal to `scenarioWithBasicTypes`, you probably don't need this second check